kie-api/src/build/revapi-config.json

```diff
@@ -35,7 +35,12 @@
         "code" : "java.method.addedToInterface",
         "new" : "method org.kie.api.builder.model.KieBaseModel org.kie.api.builder.model.KieModuleModel::newKieBaseModel()",
         "justification" : "Makes KieBase name optional."
+      },
+      {
+        "code" : "java.method.removed",
+        "old" : "method void org.kie.api.runtime.process.EventListener::signalEvent(java.lang.String, java.lang.Object)",
+        "justification" : "The .signalEvent(String, Object) method has been moved to the (parent) EventSignallable interface."
```

Comment:
I am completely fine with leaving it as is. It's probably even better for possible reader to see how the API changed over time.